packages/messaging/android/src/main/java/com/adobe/marketing/mobile/reactnative/messaging/RCTAEPMessagingModule.java

```diff
@@ -84,7 +86,11 @@ public void getCachedMessages(final Promise promise) {
 
   @ReactMethod
   public void getLatestMessage(final Promise promise) {
-    promise.resolve(this.latestMessage);
+    if (this.latestMessage != null) {
+      promise.resolve(RCTAEPMessagingUtil.convertToReadableMap(convertMessageToMap(this.latestMessage)));
```

Comment:
Let's remove `RCTAEPMessagingUtil.` since we're using the "static import" for the method.